app/src/main/res/values-fr/strings.xml

```diff
@@ -0,0 +1,86 @@
+    <string name="about">À propos</string>
+
+    <string name="compass_rose_image_description">Rose des vents indiquant le nord magnétique</string>
+    <string name="device_heading_indicator_image_description">Indique dans quelle direction l'appareil est orienté</string>
+
+    <string name="status_degrees_text_description">Degrées de la boussole</string>
+    <string name="status_cardinal_direction_text_description">Direction cardinale de la boussole</string>
+
+    <string name="cardinal_direction_north">Nord</string>
+    <string name="cardinal_direction_northeast">Nord-Est</string>
+    <string name="cardinal_direction_east">Est</string>
+    <string name="cardinal_direction_southeast">Sud-Est</string>
+    <string name="cardinal_direction_south">Sud</string>
+    <string name="cardinal_direction_southwest">Sud-Ouest</string>
+    <string name="cardinal_direction_west">Ouest</string>
+    <string name="cardinal_direction_northwest">Nord-Ouest</string>
+
+    <string name="cardinal_direction_north_abbreviation">N</string>
+    <string name="cardinal_direction_east_abbreviation">E</string>
+    <string name="cardinal_direction_south_abbreviation">S</string>
+    <string name="cardinal_direction_west_abbreviation">O</string>
+
+    <string name="ok">OK</string>
+
+    <string name="version">Version %s</string>
+    <string name="version_description">Nom de la version</string>
+    <string name="copyright">Copyright @ 2022 <a href="mailto:philipp.bobek@mailbox.org">Philipp Bobek</a></string>
+    <string name="copyright_description">Avis de copyright</string>
+    <string name="license">Licence <a href="https://www.gnu.org/licenses/gpl-3.0.txt">GNU GPLv3</a></string>
+    <string name="license_description">Licence du logiciel</string>
+    <string name="source_code"><a href="https://github.com/Kr0oked/Compass">Code source</a></string>
+    <string name="source_code_description">Lien vers la gestion du code source</string>
+
+	<string name="sensor_accuracy_no_contact">Capteur non disponible</string>
+    <string name="sensor_accuracy_unreliable">Valeurs des capteurs non fiables</string>
+    <string name="sensor_accuracy_low">Précision faible</string>
+    <string name="sensor_accuracy_medium">Précision moyenne</string>
+    <string name="sensor_accuracy_high">Précision maximale</string>
+    <string name="sensor_accuracy_image_description">Affichage graphique de la précision du capteur</string>
+    <string name="sensor_accuracy_text_description">Décrit la précsion du capteur</string>
+    <string name="sensor_error_message">Capteur non disponible</string>
+	<string name="sensor_calibration_text">Vous pouvez améliorer la précision de votre capteur à n'importe quel moment en tournant l'appareil en faisant un 8.</string>
+    <string name="sensor_calibration_text_description">Explique comment améliorer la précision du capteur</string>
+
+    <string name="zero_degrees" translatable="false">0°</string>
```

Comment:
You can remove these untranslateable strings from this file

Reply:
ok, thanks, didn't see that